ZPerl/Libs/LibClassicDurations/core.lua

```diff
@@ -17,7 +17,7 @@ Usage example 1:
 	end
 
 --]================]
-if WOW_PROJECT_ID ~= WOW_PROJECT_CLASSIC then return end
+if WOW_PROJECT_ID >= WOW_PROJECT_CLASSIC then return end
```

Comment:
Wrong logic.  This is supposed to return if not classic.  You just made it return IF classic.

I would go with
```
if WOW_PROJECT_ID == WOW_PROJECT_MAINLINE then return end
```

However, if this is a classic ONLY thing, then return it to the original ~= WOW_PROJECTCLASSIC (ie. I saw you removed the usage of this later in ZPerl_Target/ZPerl_Target.lua)